doc/user/backup_config.md

```diff
@@ -76,7 +82,7 @@ Then we could start using S3 storage for backups. See [spec examples](spec_examp
 
 See the [S3 backup with cluster specific configuration](https://github.com/coreos/etcd-operator/blob/master/doc/user/spec_examples.md#s3-backup-and-cluster-specific-s3-configuration) spec to see what the cluster's `spec.backup` field should be configured as to set a cluster specific S3 backup configuration. The following additional fields need to be set under the cluster spec's `spec.backup.s3` field:
 - `s3Bucket`: The name of the S3 bucket to store backups in.
-- `awsSecret`: The secret object name which should contain two files named `credentials` and `config` .
+- `awsSecret`: The secret object name which may contain two files named `credentials` and `config` .
```

Comment:
There should be more detailed explanation for how this works.
But let's move this PR forward quickly.
Maybe you can undo the changes in this file. And then submit another PR for docs change (we don't need to wait CI). Or just leave it to us to change the docs.